### ISSUE TITLE
Aperture not moving in for rotation

### ISSUE THREAD

**Issue by DominicOram** [DiamondLightSource/hyperion#1534]
During testing today we found that an aperture was not being selected for the rotation scan and the aperture/scatterguard was staying in `ROBOT_LOAD`. 

This may have been because we had no beam and so the XRC was not returning a sensible aperture to go to. Even so we should probably pick some default aperture (which we may have to do in the GDA side). 

## Acceptance Criteria
* We move to a aperture/scatterguard position before rotation

**Comment by DominicOram** [comment]
Use large as a sensible default

**Comment by olliesilvester** [comment]
XRC returned null for aperture position due to there being no beam. Hyperion defaults to not moving the aperture during rotations in this instance, but we can change it to default to large instead

**Comment by olliesilvester** [comment]
I seem to remember not moving the aperture was an intentional decision though. Do we still want the option for rotation to not move the aperture?

**Comment by DominicOram** [comment]
Yes, the decision was that for every parameter you could specify `None` to mean just give me whatever is already set. In doing the aperture/scatterguard work this has also messed me up and so I think it's just additional logic that causes confusion and we're not using. Propose we default it `Large` on the hyperion side and not change the GDA code here. Thoughts @dperl-dls?

**Comment by dperl-dls** [comment]
sounds fine to me, really should ask Neil what the default should be but large seems sensiblee

**Comment by DominicOram** [comment]
I believe we did ask at the last planning meeting and the answer was `Large` but we can ask again (@neil-i03?). I can put the default in as part of https://github.com/DiamondLightSource/hyperion/issues/1266 as it cleans that up too

**Comment by olliesilvester** [comment]
Ah, I was just about to do a PR for https://github.com/DiamondLightSource/hyperion/tree/1534_large_default_aperture
